src/engraving/dom/dynamic.cpp

```diff
@@ -255,15 +269,21 @@ void Dynamic::setDynamicType(const String& tag)
 std::pair<DynamicType, String> Dynamic::parseDynamicText(const String& tag) const
 {
     std::string utf8Tag = tag.toStdString();
-    const std::regex dynamicRegex(R"((?:<sym>.*?</sym>)+|(?:\b)[fmnprsz]+(?:\b(?=[^>]|$)))");
+    const std::regex dynamicRegex(R"((<plainDyn>.*?</plainDyn>)|(?:<sym>.*?</sym>)+|(?:\b)[fmnprsz]+(?:\b(?=[^>]|$)))");
```

Comment:
You should look for `<sym>` (or `<dyn>`) tags, not `<plainDyn>` tags.

If your style option is set and you find a `<sym>` (or `<dyn>`) tag that corresponds to a dynamic, then you know it must be a plain dynamic even without the `<plainDyn>` tag.